device-types/Dell/PowerConnect-3548p.yaml

```diff
@@ -0,0 +1,219 @@
+---
+manufacturer: Dell
+model: PowerConnect 3548p
+slug: dell-powerconnect-3548p
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: rs-232
+    type: de-9
+power-ports:
+  - name: Power
```

Comment:
Does the device actually report the power is named `Power`? We want to make sure that the interface names match what the configuration/device reports them as

Reply:
I updated this - according to the docs the name is "PS1", and it has a redundant power supply / extended power supply port, so I went ahead and added that as well.

Reply:
Great, thanks!